setup.py

```diff
@@ -14,6 +14,8 @@
     'cloudpickle',
     # Deprecation management
     'deprecation>=2',
+    # Progress bars
+    'tqdm>=4.10.0',
```

Comment:
```suggestion
    'tqdm~=4.10.0',
```
See comment for `requirements.txt` above.